jobs/payment-jobs/pyproject.toml

```diff
@@ -27,6 +27,7 @@ launchdarkly-server-sdk = "^8.2.1"
 more-itertools = "^10.2.0"
 cloud-sql-python-connector = "^1.13.0"
 ruff = "^0.14.1"
+pg8000 = "^1.31.5"
```

Comment:
Need for DW job to sync BCOL refund invoices to final status  cloud-sql-connector doesn't support psycopg